bus/routes.go

```diff
@@ -470,22 +469,22 @@ func (b *Bus) walletPrepareFormHandler(jc jape.Context) {
 		jc.Error(errors.New("no renter key provided"), http.StatusBadRequest)
 		return
 	}
-	cs := b.cm.TipState()
```

Comment:
I'd target the `v2` branch and then remove the legacy contract formation endpoints on bus and worker. That should clean things up a bit. e.g. we don't need a `prepareForm` anymore. It can just be a `bus.FormContract` which does everything in one step without having to do anything like returning a function to discard txns.